File: Clases_Csharp/Consulta.cs
using System;

namespace SistemaVeterinaria
{
    public class Consulta
    {
        public string IdConsulta { get; private set; }
        public string Fecha { get; private set; }
        public double Peso { get; private set; }
        public double Temperatura { get; private set; }
        public string Sintomas { get; private set; }
        public string Diagnostico { get; private set; }
        public string Farmaco { get; private set; }
        public string Dosis { get; private set; }
        public string Frecuencia { get; private set; }

        public Consulta(string idConsulta, string fecha, double peso, double temperatura,
                        string sintomas, string diagnostico, string farmaco, string dosis, string frecuencia)
        {
            IdConsulta = idConsulta;
            Fecha = fecha;
            Peso = peso;
            Temperatura = temperatura;
            Sintomas = sintomas;
            Diagnostico = diagnostico;
            Farmaco = farmaco;
            Dosis = dosis;
            Frecuencia = frecuencia;
        }

        public void MostrarResumen()
        {
            Console.WriteLine("Consulta ID: " + IdConsulta);
            Console.WriteLine("Fecha: " + Fecha + " | Diagnóstico: " + Diagnostico);
            Console.WriteLine("Signos vitales -> Peso: " + Peso + "kg, Temp: " + Temperatura + "°C");
            Console.WriteLine("Síntomas: " + Sintomas);
            if (!string.IsNullOrEmpty(Farmaco))
            {
                Console.WriteLine("Receta: " + Farmaco + " | Dosis: " + Dosis + " | Frecuencia: " + Frecuencia);
            }
        }
    }
}

File: Clases-Bluej/Consulta.java
public class Consulta {
    private String idConsulta;
    private String fecha;
    private double peso;
    private double temperatura;
    private String sintomas;
    private String diagnostico;
    private String farmaco;
    private String dosis;
    private String frecuencia;

    public Consulta(String idConsulta, String fecha, double peso, double temperatura, 
                    String sintomas, String diagnostico, String farmaco, String dosis, String frecuencia) {
        this.idConsulta = idConsulta;
        this.fecha = fecha;
        this.peso = peso;
        this.temperatura = temperatura;
        this.sintomas = sintomas;
        this.diagnostico = diagnostico;
        this.farmaco = farmaco;
        this.dosis = dosis;
        this.frecuencia = frecuencia;
    }

    public void mostrarResumen() {
        System.out.println("Consulta ID: " + idConsulta);
        System.out.println("Fecha: " + fecha + " | Diagnóstico: " + diagnostico);
        System.out.println("Signos vitales -> Peso: " + peso + "kg, Temp: " + temperatura + "°C");
        System.out.println("Síntomas: " + sintomas);
        if (farmaco != null && !farmaco.isEmpty()) {
            System.out.println("Receta: " + farmaco + " | Dosis: " + dosis + " | Frecuencia: " + frecuencia);
        }
    }

    public String getIdConsulta() { return idConsulta; }
    public String getFecha() { return fecha; }
    public String getDiagnostico() { return diagnostico; }
}